docs/important/relevant_functionalities_en.md

```diff
@@ -1044,30 +1044,85 @@ This script is executed before the upgrade process and should only output inform
 
 This chapter describes features that integrate Dogus deeper into the Cloudogu EcoSystem and make them uniformly administrable.
 
-### Memory/Swap Limit
+### Resource requirements
 
-Memory and swap limits can be used to restrict the memory consumption (working memory and swap space) of Dogus.
+The resource requirements for a Dogu can be applied to the following resources:
 
-If a Dogu exceeds its memory limit, the largest process in the container is killed. This is usually the main process of the Dogus and causes the container to be restarted.
+* Memory (RAM)
+* Swap
+* CPU cores
+* Ephemeral storage
 
-If no value is set for memory limiting, it will not take place. For swap limiting `0b` is the default value and thus does not provide swap.
+A distinction is made between resource requests and limits:
 
-#### Preparation of the memory limits in the host
+- **Resource Requests:** Specify the minimum resources (CPU cores, memory, ephemeral storage) required by a dogu for it to be functional.
+  In the multinode EcoSystem, the Kubernetes scheduler ensures that the Dogu is started on a node with sufficient resources.
+- **Resource Limits:** Specify the maximum amount of resources a Dogu is allowed to use.
 
-In order for the Cloudogu EcoSystem host to limit memory **and** swap, the following settings must be done beforehand:
-1. open the file `/etc/default/grub`
-2. add the following value to the variable `GRUB_CMDLINE_LINUX_DEFAULT`:
-   `cgroup_enable=memory swapaccount=1`
+#### Memory
+
+Memory limits can be used to restrict the memory consumption (RAM) of Dogus.
+
+If a Dogu exceeds its memory limit, the largest process in the container is terminated.
+This is usually the main process of the Dogus and causes the container to be restarted.
+If no value is set for the memory limit, this will not happen.
+
+Memory requests can be used to specify the minimum memory requirement for Dogus to be fully functional.
+
+**Caution!**
+Memory-Requests are only used in the Multinode-EcoSystem.
+
+#### Swap
+
+Swap limits can be used to restrict the memory consumption (swap space) of Dogus.
+
+If a Dogu exceeds its swap limit, the largest process in the container is terminated.
+This is usually the main process of the Dogus and causes the container to be restarted.
+If no value is set on swap limiting, it will not take place.
+For swap limiting `0b` is the default value and thus does not provide swap.
+
+**Caution!**
+Swap limits are only applied in the Singlenode-EcoSystem!
```

Comment:
Maybe we invert this sentence since we only know that it doesn't work on Kubernetes. But maybe it does work on the next big thing.
```suggestion
Swap limits are **NOT** applied in the Multinode-EcoSystem!
```